dev/com.ibm.ws.beanvalidation.v20.cdi/src/com/ibm/ws/beanvalidation/v20/cdi/internal/ValidationReleasableFactoryImpl.java

```diff
@@ -274,7 +274,13 @@ private void addValueExtractorBeans(Configuration<?> config, ClassLoader appClas
         List<ValueExtractor> valueExtractors;
 
         if (System.getSecurityManager() != null) {
-            valueExtractors = AccessController.doPrivileged(GetInstancesFromServiceLoader.action(PrivHelper.getContextClassLoader(), ValueExtractor.class));
+            ClassLoader tccl = AccessController.doPrivileged(new PrivilegedAction<ClassLoader>() {
+                @Override
+                public ClassLoader run() {
+                    return Thread.currentThread().getContextClassLoader();
+                }
+            });
+            valueExtractors = AccessController.doPrivileged(GetInstancesFromServiceLoader.action(tccl, ValueExtractor.class));
```

Comment:
Why wouldn't we do both of these back-to-back operations within a single doPriv?
valueExtractors = AccessController.doPrivileged(
   ...
       return GetInstancesFromServiceLoader.action(Thread.currentThread().getContextClassLoader(), ValueExtractor.class);

Reply:
Let's update this in the next code drop we do.